src/plugins/interceptors/missingObjectInterceptor.js

```diff
@@ -38,6 +38,7 @@ export default function MissingObjectInterceptor(openmct) {
       }
 
       return object;
-    }
+    },
+    priority: openmct.priority.HIGH + 1 // this should be the first interceptor to run
```

Comment:
Could we just make the style interceptor a lower priority?

Reply:
yeah it didn't do anything either :) I just added a check in the interceptors to make sure the domain object exists